shopinvader/views/shopinvader_variant_view.xml

```diff
@@ -65,14 +72,29 @@
         <field name="arch" type="xml">
             <form>
                 <sheet>
-                    <group name="variant">
-                        <field name="name" />
-                        <field name="shopinvader_product_id" />
-                        <field name="lang_id" />
-                        <field name="default_code" />
-                        <field name="backend_id" />
-                        <field name="tmpl_record_id" />
-                        <field name="record_id" />
+                    <field name="active" invisible="1" />
+                    <widget
+                        name="web_ribbon"
+                        text="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" class="oe_inline" readonly="1" />
+                        </h1>
+                    </div>
+                    <group>
+                        <group name="backend">
+                            <field name="backend_id" />
+                            <field name="lang_id" />
+                            <field name="shopinvader_product_id" />
+                        </group>
+                        <group name="variant">
+                            <field name="default_code" />
+                            <field name="tmpl_record_id" invisible="1" />
```

Comment:
I'd not hide this field. It's very handy to jump directly to the real record. I would do this:

1. add a separted group on the right named "Real records" or something similar
2. move there all the references to original records

Reply:
Yes indeed, I did the same with the other forms `<group name="record">`, but here I didn't want to change the group name as it already existed and it may be used by other modules in their xpath target.

Note that I'm only hiding the `tmpl_record_id` which is not `record_id` (the actual binding)

Reply:
yeah, my take is that is very handy to be able to jump directly to the real template instead of passing thru other records

Reply:
Done !